render/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlLiteralSerializer.kt

```diff
@@ -44,6 +45,12 @@ class JpqlLiteralSerializer : JpqlSerializer<JpqlLiteral<*>> {
         writer.write(part.boolean)
     }
 
+    private fun serialize(part: JpqlLiteral.CharLiteral, writer: JpqlWriter) {
+        writer.write("'")
+        writer.write("${part.char}")
+        writer.write("'")
```

Comment:
Since single quote can come in as char, it need code to convert one single quote to two single quotes.